scripts/generateBandageLabelsFromAlignment.py

```diff
@@ -0,0 +1,117 @@
+from collections import defaultdict
+import argparse
+
+
+def getChromosomeColor(refName):
+    if refName == "chr1":
+        return "sea green"
+    elif refName == "chr2":
+        return "tomato"
+    elif refName == "chr3":
+        return "dark orange"
+    elif refName == "chr4":
+        return "dark orchid"
+    elif refName == "chr5":
+        return "firebrick"
+    elif refName == "chr6":
+        return "deep pink"
+    elif refName == "chr7":
+        return "#800000"
+    elif refName == "chr8":
+        return "web green"
+    elif refName == "chr9":
+        return "green yellow"
+    elif refName == "chr10":
+        return "gold"
+    elif refName == "chr11":
+        return "midnight blue"
+    elif refName == "chr12":
+        return "light coral"
+    elif refName == "chr13":
+        return "light green"
+    elif refName == "chr14":
+        return "red"
+    elif refName == "chr15":
+        return "aquamarine"
+    elif refName == "chr16":
+        return "dodger blue"
+    elif refName == "chr17":
+        return "orange red"
+    elif refName == "chr18":
+        return "yellow"
+    elif refName == "chr19":
+        return "teal"
+    elif refName == "chr20":
+        return "aqua"
+    elif refName == "chr21":
+        return "hot pink"
+    elif refName == "chr22":
+        return "light salmon"
+    elif refName == "chrX":
+        return "yellow green"
+    elif refName == "chrY":
+        return "olive drab"
+    else:
+        print("Warning using gray for unknown chromosome: " + refName)
+        return "gray"
```

Comment:
Instead of returning gray, consider generating a color in HSL space with random hue and fixed saturation/lightness. That would make the script more usable for non-human genomes, or for human genomes where chromosomes are labeled differently.

Reply:
In the end I disallowed all non-conventional chromosome names on line 70 during the counting loop because the decoy/alt chromosomes in the reference were annoyingly attracting a lot of alignments